### ISSUE TITLE
AlexNet

### ISSUE THREAD

**Issue by Banittos** [natanielruiz/deep-head-pose#36]
Hello,
First, I am very impressed by the work, the use of regression loss on softmax output is very none-intuitive but neat and very effective.
I am interested in testing skinnier versions of you model.
In the paper you have stated two results on AlexNet in table 4 & 5 (alpha=1) what is the difference between them that caused such different results?
In the github you have the AlexNet model for testing. Can you also provide a pre-trained snapshot (weights) please?

Also I've notice on the issues you said that ResNet34 also worked nice. You have code and snapshot ready for that?

Thanks a lot!!

**Comment by natanielruiz** [comment]
Hi, thanks so much for using the work :)

I think you shouldn't use AlexNet, it doesn't seem to get much improvement from the classification+regression loss.

I would use networks with skip connections such as ResNets if I were you, and yes ResNet34 works ass well as ResNet50.

I don't have the trained snapshots on me, sorry.
